site/docs/row-level-deletes.md

```diff
@@ -0,0 +1,190 @@
+<!--
```

Comment:
```suggestion
---
url: row-level-deletes
aliases:
    - "tables/row-level-deletes"
---
```
This will need this front-matter added to the top and the file should be relocated to the [docs/tables](https://github.com/apache/iceberg/tree/master/docs/tables) directory.